README.md

```diff
@@ -15,6 +15,21 @@ The library provides two main functionalities:
 The [documentation](https://craft-cli.readthedocs.io) is available in Read The Docs.
 
 
+# Setting up the environment
+
+Install at system level:
+
+    sudo snap install pyright
+
+Create a virtual environment, activate it, and install developer dependencies:
+
+    python3 -m venv env
```

Comment:
Should we use a user-selectable name such as `python3 -m venv <dir name>`?

Reply:
In this case, I prefer to give instructions that you can copy and execute. Not that you need to understand how to modify to change them.